test/unit/test.rdf.js

```diff
@@ -333,6 +333,21 @@ describe("lib/rdf", function() {
       expect(firefox.childNodes[5].tagName).to.be.equal("em:maxVersion");
       expect(firefox.childNodes[5].childNodes[0].data).to.be.equal(MAX_VERSION);
     });
+
+    it("passes through raw GUIDs as targetApplication", function() {
+      var xml = setupRDF({ engines: {
+        "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}": "*"
+      }});
+      var apps = xml.getElementsByTagName("em:targetApplication");
+      var firefox = apps[0].childNodes[1]; // Description
+      expect(firefox.childNodes[1].tagName).to.be.equal("em:id");
+      expect(firefox.childNodes[1].childNodes[0].data).to.be.equal(GUIDS.FIREFOX);
+
+      expect(firefox.childNodes[3].tagName).to.be.equal("em:minVersion");
+      expect(firefox.childNodes[3].childNodes[0].data).to.be.equal(MIN_VERSION);
+      expect(firefox.childNodes[5].tagName).to.be.equal("em:maxVersion");
+      expect(firefox.childNodes[5].childNodes[0].data).to.be.equal(MAX_VERSION);
```

Comment:
for this test, I don't think it's important to make any assertions about minVersion or maxVersion. Those are covered in other tests.